modin/tests/pandas/dataframe/test_map_metadata.py

```diff
@@ -1797,6 +1799,13 @@ def test_constructor(data):
     df_equals(pandas_df, modin_df)
 
 
+def test_pyarrow_constructor():
+    pa = pytest.importorskip("pyarrow")
+
+    data = [[Decimal("3.19"), None], [None, Decimal("-1.23")]]
+    df_equals(*create_test_dfs(data, dtype=pd.ArrowDtype(pa.decimal128(3, scale=2))))
```

Comment:
`df_equals` function is used specifically since there is no benefit to using `eval_general` function (because the results of the constructors are tested).